upgrade_tests/storage_engine_upgrade_test.py

```diff
@@ -69,7 +69,7 @@ def _do_upgrade(self, login_keyspace=True):
         node1.set_install_dir(install_dir=self.fixture_dtest_setup.default_install_dir)
         self.install_legacy_parsing(node1)
         node1.start(wait_for_binary_proto=True)
-        if self.dtest_config.cassandra_version_from_build >= MAJOR_VERSION_5:
+        if node1.get_cassandra_version() >= '5.1':
```

Comment:
This seems to be the cause of all the failures in CASSANDRA-19099, we missed this when bumping TCM from 5.0 to 5.1.

Reply:
👍